frontend/micro-ui/web/micro-ui-internals/packages/modules/hearings/src/pages/employee/InsideHearingMainPage.js

```diff
@@ -1,27 +1,204 @@
-import React from "react";
-import { useTranslation } from "react-i18next";
-import { useHistory, Link } from "react-router-dom";
+import React, { useEffect, useRef, useState } from "react";
+import { useHistory } from "react-router-dom";
+import { useQuery, useQueryClient } from "react-query";
 import { Header, ActionBar, SVG, SubmitBar, Card } from "@egovernments/digit-ui-react-components";
 import { Button, TextArea } from "@egovernments/digit-ui-components";
 import EvidenceHearingHeader from "./EvidenceHeader";
 import HearingSideCard from "./HearingSideCard";
+import debounce from "lodash/debounce";
+import useGetHearings from "../../../../dristi/src/hooks/dristi/useGetHearings";
```

Comment:
don't import cross modules, across modules you have to get the hooks from Digit.Hooks.<hook-name>